modules/core/02-client/client/cli/tx.go

```diff
@@ -241,6 +245,62 @@ func NewUpgradeClientCmd() *cobra.Command {
 	return cmd
 }
 
+// NewCmdRecoverClientProposal defines the command to recover an IBC light client
+func NewCmdRecoverClientProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "recover [subject-client-id] [substitute-client-id] [flags]",
+		Args:  cobra.ExactArgs(2),
+		Short: "recover an IBC client",
+		Long: "Submit a recover IBC client proposal along with an initial deposit.\n" +
+			"Please specify a subject client identifier you want to recover..\n" +
+			"Please specify the substitute client the subject client will be recovered to.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			proposal, err := govcli.ReadGovPropFlags(clientCtx, cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			subjectClientID := args[0]
+			substituteClientID := args[1]
+
+			authority, _ := cmd.Flags().GetString(FlagAuthority)
```

Comment:
can't the `authority` (i.e. signer) be taken from the `--from` flag?

Reply:
`cmd.Flags().GetString(flags.FlagFrom)` where `flags` is "github.com/cosmos/cosmos-sdk/client/flags"

Reply:
I would assume so, this follows what sdk is up  to, can switch around if you think it would be okay to do so

Reply:
fine by me to keep this pattern

Reply:
and all was good and swell in the land of ibc-go reviews